futures-util/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+//! Combinators and utilities for working with futures-rs `Futures`s, `Stream`s, and `Sink`s.
+
+#![no_std]
+#![deny(missing_docs, missing_debug_implementations)]
+#![doc(html_root_url = "https://docs.rs/futures/0.1")]
+
+#[macro_use]
+extern crate futures_core;
+extern crate futures_executor;
+extern crate futures_sink;
+
+use futures_core::{Async, Future, IntoFuture, Poll, Stream};
+use futures_core::task;
+use futures_sink::{AsyncSink, Sink, StartSend};
+
+#[macro_use]
+#[cfg(feature = "std")]
+extern crate std;
+
+#[cfg(feature = "std")]
+pub mod lock;
```

Comment:
I think it's probably ok to avoid exporting this module, right?

Reply:
This is the old `bilock` stuff that was under `sync`. I'm fine w/ leaving it out if you are.

Reply:
Oh oops! Yeah seems fine to leave here.